### PR TITLE
[Monitor]add example for diagnostic-settings categories list

### LINKED ISSUE #10744: The article doesn't list any samples.
Please refer this page. It has all the details required with examples :
https://docs.microsoft.com/en-us/rest/api/monitor/diagnosticsettingscategory/list

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 55b103cb-dc59-8949-663f-f968a48d204e
* Version Independent ID: 387ce519-f57b-e817-6046-08552966744a
* Content: [az monitor diagnostic-settings categories](https://docs.microsoft.com/en-us/cli/azure/monitor/diagnostic-settings/categories?view=azure-cli-latest)
* Content Source: [src/azure-cli/azure/cli/command_modules/monitor/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/monitor/_help.py)
* Service: **monitoring-and-diagnostics**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by MyronFanQiu**
Fix #10744
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/monitor/_help.py [hunk 1]
```diff
@@ -748,6 +748,17 @@
 short-summary: Retrieve service diagnostic settings categories.
 """
 
+helps['monitor diagnostic-settings categories list'] = """
+type: command
+short-summary:  Lists the diagnostic settings categories for the specified resource.
```
Comment (bim-msft) [inline src/azure-cli/azure/cli/command_modules/monitor/_help.py]: Hi, I'm confused by the verb form. Here you use the third person singular form `lists`, also I saw lots of base form in other commands such as `create`, `update`, `manage`. Should we have a consistent form in the document?

Comment (MyronFanQiu) [reply src/azure-cli/azure/cli/command_modules/monitor/_help.py]: Good point.

File: src/azure-cli/azure/cli/command_modules/monitor/_help.py [hunk 2]
```diff
@@ -748,6 +748,17 @@
...
+examples:
+  - name: List diagnostic settings categories by using resource ID
+    text: az monitor diagnostic-settings categories list --resource /subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/myRG/providers/microsoft.logic/workflows/myworkflow
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/monitor/_help.py]: I saw myResource and MyResource. Any convention?

Comment (MyronFanQiu) [reply src/azure-cli/azure/cli/command_modules/monitor/_help.py]: In my opinion, it should be myResource. I will double check this PR.

**Pull request merged**